src/screens/LocationScreen.js

```diff
@@ -126,7 +125,7 @@ class LocationScreen extends Component {
       <LinearGradient colors={["#94e4f9", "#2d93d8"]} style={styles.container}>
         <Text style={styles.title}>Guardian</Text>
         <Image
-          source={require("./images/logo.png")}
+          source={require("../../CovidGuardianClient/images/logo.png")}
```

Comment:
these images should move to the top level directory, `CovidGuardianClient` should no longer exist

Reply:
Above, Luxx suggested we move these to App.js loadResourcesAsync. I will inquire about this in Discord.